docker/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.7'
+services:
+  notebook:
+    image: ohm314/nmodl:0.2
+    build: recipe
+    hostname: ${HOSTNAME}
+    ports:
+    - "8888:8888"
+    environment:
+    - USER_LOGIN=${USER}
+    - USER_ID=${DUID}
+    - GROUP_ID=${DGID}
+    volumes:
+    - $PWD/notebooks:/nmodl/notebooks/my_notebooks
+    command:
```

Comment:
The `command` argument is not required as already specified in the docker image with the `CMD` directive.

Reply:
I was wondering about that. But maybe I’ll leave it for clarity. The end-user can easily see what is run without having to look into the Dockerfile